bin/src/runRollup.js

```diff
@@ -217,7 +216,7 @@ function bundle ( options ) {
 		handleError({ code: 'MISSING_INPUT_OPTION' });
 	}
 
-	return rollup.rollup( options ).then( bundle => {
+	return rollup( options ).then( bundle => {
```

Comment:
If you import the namespace `rollup = require('rollup')` or `import * as rollup ...` then you have to do `rollup.rollup`, but if you import it directly (`import { rollup } ...`) then you don't – I guess it's just a matter of personal preference. I'd generally expect people to use `rollup.rollup` in CommonJS land and `rollup` in ES module land (same as `fs.readFile(...)` as opposed to `readFile(...)`) but it's a matter of preference.

The `rollup.rollup` duplication was just me amusing myself, I got a kick out of imagining everyone trying to figure out why the [Magical Mystery Tour](https://www.youtube.com/watch?v=kTOyVcs2jkI) was going round and round in their head while they were coding...